apps/extension/src/state/connected-sites.test.ts

```diff
@@ -19,6 +19,7 @@ describe('Connected Sites Slice', () => {
 
   beforeEach(() => {
     localStorage = mockLocalExtStorage();
+    //
     useStore = create<AllSlices>()(initializeStore(mockSessionExtStorage(), localStorage));
```

Comment:
Here and in a number of other tests I get an error 
```typescript
TS2322: Type 'UseBoundStore<WithImmer<StoreApi<AllSlices>>>' is not assignable to type 'UseBoundStore<StoreApi<AllSlices>>'.
``` 

<img width="1438" alt="image" src="https://github.com/penumbra-zone/web/assets/25391690/9477bc62-d5fe-497b-972d-1c1201bf3876">

Reply:
So this is a downside of zustand+immer 😞 . It's not allowed to store bufbuild classes directly. It seems to be a limitation because immer cannot properly do a safe copy of it. So we likely need to be using `WalletJson` here. But does this invalidate this whole task? Maybe not, we still should be storing FullViewingKey.toJson() in storage instead of a string.